mappings.json

```diff
@@ -881,12 +881,32 @@
     "global": "Ember.String.htmlSafe",
     "module": "@ember/string",
     "export": "htmlSafe",
+    "deprecated": true,
+    "replacement": {
+      "module": "@ember/template",
+      "export": "htmlSafe"
+    }
+  },
+  {
+    "global": "Ember._Template.htmlSafe",
```

Comment:
Hmm, I think this needs to stay `Ember.String.htmlsafe` until we have a way to ensure that the transpilation will work for both ember-source@2.16 and ember-source@3.0.0-beta.1 (soon to include this global).

Thoughts?

Reply:
something like `["Ember._Template.htmlSafe", "Ember.String.htmlSafe"]`?

but if we're going to make the format more complicated we should definitely document the format better 🤔

Reply:
I would change it and think how to do different transpilations depending on the version. Keeping an array would force us to transpile to something like:

```js
var x = (Ember._Template && Ember._Template.htmlSafe) || (Ember.String && Ember.String.htmlSafe);
```

I rather keep two versions (0.x for Ember.String and 0.(x+y) for Ember._Template) than start doing that. In case we want to keep all in one version, I would rather see something like:

```json
{
  "global": [{ "version_selector": "<2.18.0", "global": "Ember.String.htmlSafe"}, { "version_selector": ">=2.18.0", "global": "Ember._Template.htmlSafe"}]
}
```